fluidBuild.config.cjs

```diff
@@ -243,7 +243,9 @@ module.exports = {
 				"^tools/getkeys",
 			],
 			"npm-package-json-esm": [
-				"packages/dds/tree/package.json", // Policy is incorrect about "module" in package.json
+				// Policy is incorrect about "module" in package.json
+				"experimental/PropertyDDS/packages/property-shared-tree-interop/package.json",
+				"packages/dds/tree/package.json",
```

Comment:
Will we need to add an entry for every package we convert to '"type": "module"'?